.circleci/config.yml

```diff
@@ -32,24 +38,10 @@ jobs:
             sudo npm install -g markdown-link-check
             bash .circleci/check-markdown-links.sh
 
-  check_code:
-    executor: ubuntu_vm
-    steps:
-      - checkout
-
-      # If the only changes are to the docs, don't run the slow (and expensive) code checks
-      # `git diff --exit-code` returns 0 exit code if no changes, and 1 if changes
-      - run: |
-          if git --no-pager diff --exit-code origin/develop...HEAD -- . ':!docs'; then
```

Comment:
Why did you remove this control? Is it not a good control in order to avoid execute validate-boostrap if there are not changes there?